src/v2/components/Layout/Layout.tsx

```diff
@@ -0,0 +1,76 @@
+import { ReactNode } from "react"
+import { PropsWithChildren } from "react"
+import { PageBoundary } from "@/components/PageBoundary"
+// import { Header as NavHeader } from "@/components/Header"
+const NavHeader = () => <div className="h-16 border-border border-b">mocked header</div>
+
+/* -------------------------------------------------------------------------------------------------
+ * Root
+ * -----------------------------------------------------------------------------------------------*/
+
+const Root = ({ children }: PropsWithChildren) => (
+  <div className="flex min-h-screen flex-col">{children}</div>
+)
+
+/* -------------------------------------------------------------------------------------------------
+ * Header
+ * -----------------------------------------------------------------------------------------------*/
+
+const Header = ({ children }: PropsWithChildren) => (
+  <header className="relative">
+    <NavHeader />
+    {children}
+  </header>
+)
+
+/* -------------------------------------------------------------------------------------------------
+ * Headline
+ * -----------------------------------------------------------------------------------------------*/
+
+interface HeadlineProps {
+  title: ReactNode
+}
+
+const Headline = ({ title }: HeadlineProps) => (
+  <PageBoundary>
+    <h1 className="pb-14 pt-9 font-display text-4xl font-bold tracking-tight text-white sm:text-5xl">
+      {title}
+    </h1>
+  </PageBoundary>
+)
+
+/* -------------------------------------------------------------------------------------------------
+ * Banner
+ * -----------------------------------------------------------------------------------------------*/
+
+const Banner = () => (
```

Comment:
This component should only handle the size & position (and it could have a default background color) & we should pass the customization related stuff to it where we actually use it ;)